src/Task/Rector.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Fixer\Provider\FixableProcessResultProvider;
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Runner\TaskResultInterface;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Process\Process;
+
+class Rector extends AbstractExternalTask
+{
+    public static function getConfigurableOptions(): OptionsResolver
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'config' => 'rector.php',
+            'triggered_by' => ['php'],
+            'ignore_patterns' => [],
+            'clear-cache' => true,
```

Comment:
grumphp uses snake_case for all its options.
This makes the configuration of multiple tasks more consistent

Reply:
Refactored the options to use snake_case 🐍